sched/Kconfig

```diff
@@ -1058,12 +1058,12 @@ config SDCLONE_DISABLE
 	descriptors by task_create() when a new task is started. If
 	set, all sockets will appear to be closed in the new task.
 
-config NFILE_DESCRIPTORS
-	int "Maximum number of file descriptors per task"
-	default 16
+config NFCHUNK_DESCRIPTORS
```

Comment:
I think the name NFCHUNK_DESCRIPTORS is enigmatic, why not NFILE_DESCRIPTORS_ROW or something similar?

Reply:
the  mean isn't NFILE_DESCRIPTORS_ROW, you can see more in #2834. we need  config the size of each chunk in the structure of  dobule fd array. But, this only my suggest.

Reply:
This option control how many file handles to extend in each reallocation, NFILE_DESCRIPTORS_ROW obviously isn't a good name, @acassis do you have other better suggestion?

Reply:
Will `NFILE_DESCRIPTORS_PER_ALLOC` work?

Reply:
NFILE_DESCRIPTORS_PER_BLOCK?
NFILE_DESCRIPTORS_PER_CHUNK?

Reply:
PER_BLOCK/PER_CHUNK looks better, @davids5 @acassis what's your opinion?

Reply:
+1 for NFILE_DESCRIPTORS_PER_BLOCK